### DISCUSSION TITLE
GUI display values

### DISCUSSION THREAD

**Discussion by shadowpulse-alt** [KlipperScreen/KlipperScreen#973] [Ideas]
It would be handy to be able to display the following values on the KlipperScreen GUI, all with a label for each value as a line at the top not far from the nozzle and platen temperatures:
3d printer motherboard mcu temperature
raspberry pi cpu and ram usage
raspberry pi cpu temperature

**Comment by alfrix** [comment]
you can add items in titlebar items as described in the config 
https://klipperscreen.readthedocs.io/en/latest/Configuration/#printer-options
raspberry pi cpu and ram usage are not supported at the moment and there is already an FR in place

**Reply by shadowpulse-alt** [reply]
Is it normal that my file is empty like this?

[image-2023-05-26-205134431.png](https://postimg.cc/0K00Gtwm)
